lib/Donors/utils.js

```diff
@@ -0,0 +1,36 @@
+import {
+  Button,
+  Icon,
+  TextLink,
+} from '@folio/stripes/components';
+
+const getDonorUrl = (orgId) => {
+  if (orgId) {
+    return `/organizations/view/${orgId}`;
+  }
+
+  return undefined;
+};
+
+export const getResultsFormatter = ({
```

Comment:
As far as you export `*` from `Donors`, the name of the function has to be more specific.

Reply:
`getDonorResultsFormatter` ?

Reply:
or `getDonorsListFormatter`